prov/efa/src/efa_device.h

```diff
@@ -21,6 +21,7 @@ struct efa_device {
 	struct ibv_pd		*ibv_pd;
 	struct fi_info		*rdm_info;
 	struct fi_info		*dgram_info;
+	struct fi_info     *direct_info;
```

Comment:
indent, also I don't think direct_info should be a separate field in `efa_device`.

Reply:
Discussed offline. Refactoring to move logic to user_info will be done later.